setup.py

```diff
@@ -114,6 +118,9 @@
     if arg.startswith("-wheel-name="):
```

Comment:
I think we should do:
```suggestion
    if arg.startswith("--wheel-name="):
```
But this is beyond the scope of this PR

Reply:
In that other PR we can switch to using `argparse` and have a nice help text printed also.